src/start/server.ts

```diff
@@ -74,6 +75,9 @@ async function main() {
     startAsyncMigrations(shutdownSignal.nextGroup.signal)
   }
 
+  // Attach abort signal to jwk generation runner
+  UrlSigningJwkGenerator.Init(shutdownSignal.nextGroup.signal)
```

Comment:
I don't think this is necessary because if we invoke the method that creates jobs in an HTTP handler, if the server closes, the abort signal will also be aborted